CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ project(sdrangel)
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/Modules)
 
 # disable only when needed
-set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD 20)
```

Comment:
No

Reply:
Ok! I will try fixing the m17 build using cpp14...

Reply:
cpp 17 is acceptable